Engines/Wine/Verbs/FAudio/script.js

```diff
@@ -12,6 +12,7 @@ include("utils.functions.filesystem.files");
 Wine.prototype.faudio = function (faudioVersion) {
     if (this.architecture() != "amd64")
         this.wizard().message(tr("FAudio does not support 32bit architecture."));
+        break;
```

Comment:
You can use throw also
```
throw ""FAudio does not support 32bit architecture.";
```
which will stop the script and display the error in a window.